src/OFS/tests/testFileAndImage.py

```diff
@@ -302,6 +302,18 @@ def testFindFile(self):
         self.assertEqual(len(results), 1)
         self.assertEqual(results[0][1], self.file)
 
+    def testHistoryCompare(self):
+        self.file.manage_edit('a', 'text/plain',
+                              filedata='content_of_a')
+        getattr(self.app, self.factory)('b',
+                file='content_of_b', content_type='text/plain')
+        page = self.file.manage_historyCompare(
+            self.file,
+            self.app.b,
+            self.app.REQUEST)
+        self.assertTrue('content_of_a' in page)
```

Comment:
```suggestion
        self.assertIn('content_of_a', page)
```

This will result in a nicer error message if the assertion fails.

Reply:
Thanks, this is also what I did first, but travis-ci also runs on python 2.6, where `assertIn` was not yet available. ( For the records, error on my branch when I tried `assertIn` was https://travis-ci.org/perrinjerome/Zope/jobs/456256346 )

I see that on [setup.py](https://github.com/zopefoundation/Zope/blob/2.13/setup.py#L43) python 2.6 is listed, so if I understand correctly it's not only travis-ci and we have to stick this old style assertion.

So we leave it like this ?

Reply:
Sorry, you are absolutely right. I always forget that Zope 2.13 still supports Python 2.6. So please keep your code as it is.